src/NuGet.Core/NuGet.PackageManagement/Resolution/ResolverGather.cs

```diff
@@ -226,6 +226,16 @@ public int MaxDegreeOfParallelism
                     }
                 }
             }
+
+            foreach(var targetId in allPrimaryTargets)
+            {
+                var package = combinedResults.FirstOrDefault(p => StringComparer.OrdinalIgnoreCase.Equals(p.Id, targetId));
+                if(package != null)
+                {
+                    _context.Log.LogVerbose(string.Format("package '{0}' found from feed '{1}'", package.Id, package.Source.PackageSource.Source));
```

Comment:
This is verbose log which won't be there by default. so no need, besides it should be in english